src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,208 @@
+        }
+        deleteCarDrivers(id);
+        return deleted;
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String getCarsByDriverQuery = "SELECT id, model, manufacturer "
+                + "FROM cars c JOIN cars_drivers cd ON c.id = cd.car_id "
+                + "WHERE cd.driver_id = ? AND is_deleted = FALSE;";
+        List<Car> carList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getCarsByDriverQuery)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                carList.add(parseCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all car with driver id " + driverId, e);
+        }
+        for (Car car: carList) {
+            car.setDrivers(getDriversForCar(car.getId()));
+        }
+        return carList;
+    }
+
+    private Car parseCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("car_id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("manufacturer_name"));
+        manufacturer.setCountry(resultSet.getString("manufacturer_country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String getDriversForCarQuery = "SELECT id, name, license_number "
+                + "FROM drivers d JOIN cars_drivers cd ON d.id = cd.driver_id "
+                + "WHERE cd.car_id = ? AND is_deleted = FALSE;";
+        List<Driver> driverList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getDriversForCarQuery)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                driverList.add(parseDriver(resultSet));
+            }
+            return driverList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all drivers with car id " + carId, e);
+        }
+    }
+
+    private Driver parseDriver(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("id", Long.class));
+        driver.setName(resultSet.getString("name"));
+        driver.setLicenseNumber(resultSet.getString("license_number"));
+        return driver;
+    }
+
+    private void createCarDrivers(Car car) {
```

Comment:
```suggestion
    private void addDriversToCar(Car car) {
```
I think drivers are already created in db and we just connect them to car in db